revisions/N0/src/text/dialog_dx.asm

```diff
@@ -94,6 +94,7 @@ Dialog255::
 
 Dialog256::
     db ". . . . .@"
+; Denne dialogboksen ser veldig rar ut syns jeg. Kanskje midstille teksten, eller bruke engelsk ellipse?
```

Comment:
enig. to . . . eller midtstille

Reply:
jeg foreslår å bare gjøre som i dialogen til Vindfisken, å ha engelske ellipser på én linje. Sånn her ser den originale dialogen ut in-game: 
<img width="628" alt="image" src="https://github.com/tobiasvl/zelda-links-awakening-NO/assets/16557480/b036b388-cd3d-4cce-8108-03c98640aa6b">